[2024-01-30T02:53:48.180+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: manage_data_lake_dag.local_job_raw_to_formatted scheduled__2024-01-10T00:00:00+00:00 [queued]>
[2024-01-30T02:53:48.186+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: manage_data_lake_dag.local_job_raw_to_formatted scheduled__2024-01-10T00:00:00+00:00 [queued]>
[2024-01-30T02:53:48.187+0000] {taskinstance.py:2170} INFO - Starting attempt 2 of 6
[2024-01-30T02:53:48.203+0000] {taskinstance.py:2191} INFO - Executing <Task(SparkSubmitOperator): local_job_raw_to_formatted> on 2024-01-10 00:00:00+00:00
[2024-01-30T02:53:48.208+0000] {standard_task_runner.py:60} INFO - Started process 18666 to run task
[2024-01-30T02:53:48.214+0000] {standard_task_runner.py:87} INFO - Running: ['airflow', 'tasks', 'run', 'manage_data_lake_dag', 'local_job_raw_to_formatted', 'scheduled__2024-01-10T00:00:00+00:00', '--job-id', '180', '--raw', '--subdir', 'DAGS_FOLDER/data_lake_dag.py', '--cfg-path', '/tmp/tmp2k6gnow5']
[2024-01-30T02:53:48.216+0000] {standard_task_runner.py:88} INFO - Job 180: Subtask local_job_raw_to_formatted
[2024-01-30T02:53:48.259+0000] {task_command.py:423} INFO - Running <TaskInstance: manage_data_lake_dag.local_job_raw_to_formatted scheduled__2024-01-10T00:00:00+00:00 [running]> on host instance-1.europe-west9-a.c.data-lake-project-409321.internal
[2024-01-30T02:53:48.313+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='juniortemgoua0' AIRFLOW_CTX_DAG_ID='manage_data_lake_dag' AIRFLOW_CTX_TASK_ID='local_job_raw_to_formatted' AIRFLOW_CTX_EXECUTION_DATE='2024-01-10T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='2' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-01-10T00:00:00+00:00'
[2024-01-30T02:53:48.320+0000] {base.py:83} INFO - Using connection ID 'spark_default' for task execution.
[2024-01-30T02:53:48.321+0000] {spark_submit.py:401} INFO - Spark-Submit cmd: spark-submit --master local[*] --conf spark.airflow.execution_date=2024-01-10 --name arrow-spark --class RawToFormatted --queue root.default --deploy-mode client /home/juniortemgoua0/DataLake/jobs/processes/scala/spark_process/target/scala-2.12/spark_job_2.12-0.1.0.jar hdfs://localhost:9000/data-lake
[2024-01-30T02:53:52.401+0000] {spark_submit.py:571} INFO - 24/01/30 02:53:52 INFO SparkContext: Running Spark version 3.5.0
[2024-01-30T02:53:52.411+0000] {spark_submit.py:571} INFO - 24/01/30 02:53:52 INFO SparkContext: OS info Linux, 5.15.0-1049-gcp, amd64
[2024-01-30T02:53:52.412+0000] {spark_submit.py:571} INFO - 24/01/30 02:53:52 INFO SparkContext: Java version 17.0.9
[2024-01-30T02:53:52.670+0000] {spark_submit.py:571} INFO - 24/01/30 02:53:52 WARN NativeCodeLoader: Unable to load native-hadoop library for your platform... using builtin-java classes where applicable
[2024-01-30T02:53:52.997+0000] {spark_submit.py:571} INFO - 24/01/30 02:53:52 INFO ResourceUtils: ==============================================================
[2024-01-30T02:53:52.998+0000] {spark_submit.py:571} INFO - 24/01/30 02:53:52 INFO ResourceUtils: No custom resources configured for spark.driver.
[2024-01-30T02:53:53.000+0000] {spark_submit.py:571} INFO - 24/01/30 02:53:52 INFO ResourceUtils: ==============================================================
[2024-01-30T02:53:53.002+0000] {spark_submit.py:571} INFO - 24/01/30 02:53:53 INFO SparkContext: Submitted application: RawToFormatted
[2024-01-30T02:53:53.076+0000] {spark_submit.py:571} INFO - 24/01/30 02:53:53 INFO ResourceProfile: Default ResourceProfile created, executor resources: Map(cores -> name: cores, amount: 1, script: , vendor: , memory -> name: memory, amount: 1024, script: , vendor: , offHeap -> name: offHeap, amount: 0, script: , vendor: ), task resources: Map(cpus -> name: cpus, amount: 1.0)
[2024-01-30T02:53:53.089+0000] {spark_submit.py:571} INFO - 24/01/30 02:53:53 INFO ResourceProfile: Limiting resource is cpu
[2024-01-30T02:53:53.092+0000] {spark_submit.py:571} INFO - 24/01/30 02:53:53 INFO ResourceProfileManager: Added ResourceProfile id: 0
[2024-01-30T02:53:53.214+0000] {spark_submit.py:571} INFO - 24/01/30 02:53:53 INFO SecurityManager: Changing view acls to: juniortemgoua0
[2024-01-30T02:53:53.214+0000] {spark_submit.py:571} INFO - 24/01/30 02:53:53 INFO SecurityManager: Changing modify acls to: juniortemgoua0
[2024-01-30T02:53:53.214+0000] {spark_submit.py:571} INFO - 24/01/30 02:53:53 INFO SecurityManager: Changing view acls groups to:
[2024-01-30T02:53:53.215+0000] {spark_submit.py:571} INFO - 24/01/30 02:53:53 INFO SecurityManager: Changing modify acls groups to:
[2024-01-30T02:53:53.215+0000] {spark_submit.py:571} INFO - 24/01/30 02:53:53 INFO SecurityManager: SecurityManager: authentication disabled; ui acls disabled; users with view permissions: juniortemgoua0; groups with view permissions: EMPTY; users with modify permissions: juniortemgoua0; groups with modify permissions: EMPTY
[2024-01-30T02:53:53.594+0000] {spark_submit.py:571} INFO - 24/01/30 02:53:53 INFO Utils: Successfully started service 'sparkDriver' on port 37485.
[2024-01-30T02:53:53.630+0000] {spark_submit.py:571} INFO - 24/01/30 02:53:53 INFO SparkEnv: Registering MapOutputTracker
[2024-01-30T02:53:53.697+0000] {spark_submit.py:571} INFO - 24/01/30 02:53:53 INFO SparkEnv: Registering BlockManagerMaster
[2024-01-30T02:53:53.726+0000] {spark_submit.py:571} INFO - 24/01/30 02:53:53 INFO BlockManagerMasterEndpoint: Using org.apache.spark.storage.DefaultTopologyMapper for getting topology information
[2024-01-30T02:53:53.728+0000] {spark_submit.py:571} INFO - 24/01/30 02:53:53 INFO BlockManagerMasterEndpoint: BlockManagerMasterEndpoint up
[2024-01-30T02:53:53.738+0000] {spark_submit.py:571} INFO - 24/01/30 02:53:53 INFO SparkEnv: Registering BlockManagerMasterHeartbeat
[2024-01-30T02:53:53.757+0000] {spark_submit.py:571} INFO - 24/01/30 02:53:53 INFO DiskBlockManager: Created local directory at /tmp/blockmgr-9e1c9547-efe4-4303-994c-10aba87ff23c
[2024-01-30T02:53:53.783+0000] {spark_submit.py:571} INFO - 24/01/30 02:53:53 INFO MemoryStore: MemoryStore started with capacity 434.4 MiB
[2024-01-30T02:53:53.804+0000] {spark_submit.py:571} INFO - 24/01/30 02:53:53 INFO SparkEnv: Registering OutputCommitCoordinator
[2024-01-30T02:53:54.012+0000] {spark_submit.py:571} INFO - 24/01/30 02:53:54 INFO JettyUtils: Start Jetty 0.0.0.0:4040 for SparkUI
[2024-01-30T02:53:54.110+0000] {spark_submit.py:571} INFO - 24/01/30 02:53:54 INFO Utils: Successfully started service 'SparkUI' on port 4040.
[2024-01-30T02:53:54.179+0000] {spark_submit.py:571} INFO - 24/01/30 02:53:54 INFO SparkContext: Added JAR file:/home/juniortemgoua0/DataLake/jobs/processes/scala/spark_process/target/scala-2.12/spark_job_2.12-0.1.0.jar at spark://instance-1.europe-west9-a.c.data-lake-project-409321.internal:37485/jars/spark_job_2.12-0.1.0.jar with timestamp 1706583232364
[2024-01-30T02:53:54.304+0000] {spark_submit.py:571} INFO - 24/01/30 02:53:54 INFO Executor: Starting executor ID driver on host instance-1.europe-west9-a.c.data-lake-project-409321.internal
[2024-01-30T02:53:54.305+0000] {spark_submit.py:571} INFO - 24/01/30 02:53:54 INFO Executor: OS info Linux, 5.15.0-1049-gcp, amd64
[2024-01-30T02:53:54.305+0000] {spark_submit.py:571} INFO - 24/01/30 02:53:54 INFO Executor: Java version 17.0.9
[2024-01-30T02:53:54.319+0000] {spark_submit.py:571} INFO - 24/01/30 02:53:54 INFO Executor: Starting executor with user classpath (userClassPathFirst = false): ''
[2024-01-30T02:53:54.320+0000] {spark_submit.py:571} INFO - 24/01/30 02:53:54 INFO Executor: Created or updated repl class loader org.apache.spark.util.MutableURLClassLoader@135a8c6f for default.
[2024-01-30T02:53:54.381+0000] {spark_submit.py:571} INFO - 24/01/30 02:53:54 INFO Executor: Fetching spark://instance-1.europe-west9-a.c.data-lake-project-409321.internal:37485/jars/spark_job_2.12-0.1.0.jar with timestamp 1706583232364
[2024-01-30T02:53:54.462+0000] {spark_submit.py:571} INFO - 24/01/30 02:53:54 INFO TransportClientFactory: Successfully created connection to instance-1.europe-west9-a.c.data-lake-project-409321.internal/10.200.0.5:37485 after 44 ms (0 ms spent in bootstraps)
[2024-01-30T02:53:54.475+0000] {spark_submit.py:571} INFO - 24/01/30 02:53:54 INFO Utils: Fetching spark://instance-1.europe-west9-a.c.data-lake-project-409321.internal:37485/jars/spark_job_2.12-0.1.0.jar to /tmp/spark-fb5f0a52-8095-48b2-b058-e2ad3cfc7e8d/userFiles-6892f0d3-44c8-42a3-b974-221ea9c821e6/fetchFileTemp1359158817931702643.tmp
[2024-01-30T02:53:54.524+0000] {spark_submit.py:571} INFO - 24/01/30 02:53:54 INFO Executor: Adding file:/tmp/spark-fb5f0a52-8095-48b2-b058-e2ad3cfc7e8d/userFiles-6892f0d3-44c8-42a3-b974-221ea9c821e6/spark_job_2.12-0.1.0.jar to class loader default
[2024-01-30T02:53:54.537+0000] {spark_submit.py:571} INFO - 24/01/30 02:53:54 INFO Utils: Successfully started service 'org.apache.spark.network.netty.NettyBlockTransferService' on port 40151.
[2024-01-30T02:53:54.538+0000] {spark_submit.py:571} INFO - 24/01/30 02:53:54 INFO NettyBlockTransferService: Server created on instance-1.europe-west9-a.c.data-lake-project-409321.internal:40151
[2024-01-30T02:53:54.540+0000] {spark_submit.py:571} INFO - 24/01/30 02:53:54 INFO BlockManager: Using org.apache.spark.storage.RandomBlockReplicationPolicy for block replication policy
[2024-01-30T02:53:54.551+0000] {spark_submit.py:571} INFO - 24/01/30 02:53:54 INFO BlockManagerMaster: Registering BlockManager BlockManagerId(driver, instance-1.europe-west9-a.c.data-lake-project-409321.internal, 40151, None)
[2024-01-30T02:53:54.556+0000] {spark_submit.py:571} INFO - 24/01/30 02:53:54 INFO BlockManagerMasterEndpoint: Registering block manager instance-1.europe-west9-a.c.data-lake-project-409321.internal:40151 with 434.4 MiB RAM, BlockManagerId(driver, instance-1.europe-west9-a.c.data-lake-project-409321.internal, 40151, None)
[2024-01-30T02:53:54.560+0000] {spark_submit.py:571} INFO - 24/01/30 02:53:54 INFO BlockManagerMaster: Registered BlockManager BlockManagerId(driver, instance-1.europe-west9-a.c.data-lake-project-409321.internal, 40151, None)
[2024-01-30T02:53:54.561+0000] {spark_submit.py:571} INFO - 24/01/30 02:53:54 INFO BlockManager: Initialized BlockManager: BlockManagerId(driver, instance-1.europe-west9-a.c.data-lake-project-409321.internal, 40151, None)
[2024-01-30T02:53:55.604+0000] {spark_submit.py:571} INFO - 24/01/30 02:53:55 INFO SharedState: Setting hive.metastore.warehouse.dir ('null') to the value of spark.sql.warehouse.dir.
[2024-01-30T02:53:55.614+0000] {spark_submit.py:571} INFO - 24/01/30 02:53:55 INFO SharedState: Warehouse path is 'file:/home/juniortemgoua0/DataLake/spark-warehouse'.
[2024-01-30T02:53:56.608+0000] {spark_submit.py:571} INFO - ************************************************
[2024-01-30T02:53:56.609+0000] {spark_submit.py:571} INFO - Date d'exécution  : 2024-01-10
[2024-01-30T02:53:56.609+0000] {spark_submit.py:571} INFO - ************************************************
[2024-01-30T02:53:57.741+0000] {spark_submit.py:571} INFO - Exception in thread "main" org.apache.spark.sql.AnalysisException: [PATH_NOT_FOUND] Path does not exist: hdfs://localhost:9000/data-lake/raw/football_data/competitions/2024-01-10/*.json.
[2024-01-30T02:53:57.742+0000] {spark_submit.py:571} INFO - at org.apache.spark.sql.errors.QueryCompilationErrors$.dataPathNotExistError(QueryCompilationErrors.scala:1500)
[2024-01-30T02:53:57.742+0000] {spark_submit.py:571} INFO - at org.apache.spark.sql.execution.datasources.DataSource$.$anonfun$checkAndGlobPathIfNecessary$3(DataSource.scala:743)
[2024-01-30T02:53:57.742+0000] {spark_submit.py:571} INFO - at org.apache.spark.util.ThreadUtils$.$anonfun$parmap$2(ThreadUtils.scala:380)
[2024-01-30T02:53:57.742+0000] {spark_submit.py:571} INFO - at scala.concurrent.Future$.$anonfun$apply$1(Future.scala:659)
[2024-01-30T02:53:57.743+0000] {spark_submit.py:571} INFO - at scala.util.Success.$anonfun$map$1(Try.scala:255)
[2024-01-30T02:53:57.743+0000] {spark_submit.py:571} INFO - at scala.util.Success.map(Try.scala:213)
[2024-01-30T02:53:57.743+0000] {spark_submit.py:571} INFO - at scala.concurrent.Future.$anonfun$map$1(Future.scala:292)
[2024-01-30T02:53:57.743+0000] {spark_submit.py:571} INFO - at scala.concurrent.impl.Promise.liftedTree1$1(Promise.scala:33)
[2024-01-30T02:53:57.743+0000] {spark_submit.py:571} INFO - at scala.concurrent.impl.Promise.$anonfun$transform$1(Promise.scala:33)
[2024-01-30T02:53:57.744+0000] {spark_submit.py:571} INFO - at scala.concurrent.impl.CallbackRunnable.run(Promise.scala:64)
[2024-01-30T02:53:57.744+0000] {spark_submit.py:571} INFO - at java.base/java.util.concurrent.ForkJoinTask$RunnableExecuteAction.exec(ForkJoinTask.java:1395)
[2024-01-30T02:53:57.744+0000] {spark_submit.py:571} INFO - at java.base/java.util.concurrent.ForkJoinTask.doExec(ForkJoinTask.java:373)
[2024-01-30T02:53:57.744+0000] {spark_submit.py:571} INFO - at java.base/java.util.concurrent.ForkJoinPool$WorkQueue.topLevelExec(ForkJoinPool.java:1182)
[2024-01-30T02:53:57.744+0000] {spark_submit.py:571} INFO - at java.base/java.util.concurrent.ForkJoinPool.scan(ForkJoinPool.java:1655)
[2024-01-30T02:53:57.745+0000] {spark_submit.py:571} INFO - at java.base/java.util.concurrent.ForkJoinPool.runWorker(ForkJoinPool.java:1622)
[2024-01-30T02:53:57.745+0000] {spark_submit.py:571} INFO - at java.base/java.util.concurrent.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:165)
[2024-01-30T02:53:57.751+0000] {spark_submit.py:571} INFO - 24/01/30 02:53:57 INFO SparkContext: Invoking stop() from shutdown hook
[2024-01-30T02:53:57.752+0000] {spark_submit.py:571} INFO - 24/01/30 02:53:57 INFO SparkContext: SparkContext is stopping with exitCode 0.
[2024-01-30T02:53:57.769+0000] {spark_submit.py:571} INFO - 24/01/30 02:53:57 INFO SparkUI: Stopped Spark web UI at http://instance-1.europe-west9-a.c.data-lake-project-409321.internal:4040
[2024-01-30T02:53:57.786+0000] {spark_submit.py:571} INFO - 24/01/30 02:53:57 INFO MapOutputTrackerMasterEndpoint: MapOutputTrackerMasterEndpoint stopped!
[2024-01-30T02:53:57.802+0000] {spark_submit.py:571} INFO - 24/01/30 02:53:57 INFO MemoryStore: MemoryStore cleared
[2024-01-30T02:53:57.803+0000] {spark_submit.py:571} INFO - 24/01/30 02:53:57 INFO BlockManager: BlockManager stopped
[2024-01-30T02:53:57.813+0000] {spark_submit.py:571} INFO - 24/01/30 02:53:57 INFO BlockManagerMaster: BlockManagerMaster stopped
[2024-01-30T02:53:57.821+0000] {spark_submit.py:571} INFO - 24/01/30 02:53:57 INFO OutputCommitCoordinator$OutputCommitCoordinatorEndpoint: OutputCommitCoordinator stopped!
[2024-01-30T02:53:57.835+0000] {spark_submit.py:571} INFO - 24/01/30 02:53:57 INFO SparkContext: Successfully stopped SparkContext
[2024-01-30T02:53:57.837+0000] {spark_submit.py:571} INFO - 24/01/30 02:53:57 INFO ShutdownHookManager: Shutdown hook called
[2024-01-30T02:53:57.837+0000] {spark_submit.py:571} INFO - 24/01/30 02:53:57 INFO ShutdownHookManager: Deleting directory /tmp/spark-fb5f0a52-8095-48b2-b058-e2ad3cfc7e8d
[2024-01-30T02:53:57.840+0000] {spark_submit.py:571} INFO - 24/01/30 02:53:57 INFO ShutdownHookManager: Deleting directory /tmp/spark-671ca716-7b70-4666-aa4a-f7668ea89eea
[2024-01-30T02:53:57.905+0000] {taskinstance.py:2698} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/juniortemgoua0/DataLake/venv/lib/python3.8/site-packages/airflow/models/taskinstance.py", line 433, in _execute_task
    result = execute_callable(context=context, **execute_callable_kwargs)
  File "/home/juniortemgoua0/DataLake/venv/lib/python3.8/site-packages/airflow/providers/apache/spark/operators/spark_submit.py", line 174, in execute
    self._hook.submit(self._application)
  File "/home/juniortemgoua0/DataLake/venv/lib/python3.8/site-packages/airflow/providers/apache/spark/hooks/spark_submit.py", line 502, in submit
    raise AirflowException(
airflow.exceptions.AirflowException: Cannot execute: spark-submit --master local[*] --conf spark.airflow.execution_date=2024-01-10 --name arrow-spark --class RawToFormatted --queue root.default --deploy-mode client /home/juniortemgoua0/DataLake/jobs/processes/scala/spark_process/target/scala-2.12/spark_job_2.12-0.1.0.jar hdfs://localhost:9000/data-lake. Error code is: 1.
[2024-01-30T02:53:57.908+0000] {taskinstance.py:1138} INFO - Marking task as UP_FOR_RETRY. dag_id=manage_data_lake_dag, task_id=local_job_raw_to_formatted, execution_date=20240110T000000, start_date=20240130T025348, end_date=20240130T025357
[2024-01-30T02:53:57.921+0000] {standard_task_runner.py:107} ERROR - Failed to execute job 180 for task local_job_raw_to_formatted (Cannot execute: spark-submit --master local[*] --conf spark.airflow.execution_date=2024-01-10 --name arrow-spark --class RawToFormatted --queue root.default --deploy-mode client /home/juniortemgoua0/DataLake/jobs/processes/scala/spark_process/target/scala-2.12/spark_job_2.12-0.1.0.jar hdfs://localhost:9000/data-lake. Error code is: 1.; 18666)
[2024-01-30T02:53:57.934+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 1
[2024-01-30T02:53:57.949+0000] {taskinstance.py:3280} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-01-30T03:24:08.198+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: manage_data_lake_dag.local_job_raw_to_formatted scheduled__2024-01-10T00:00:00+00:00 [queued]>
[2024-01-30T03:24:08.221+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: manage_data_lake_dag.local_job_raw_to_formatted scheduled__2024-01-10T00:00:00+00:00 [queued]>
[2024-01-30T03:24:08.222+0000] {taskinstance.py:2170} INFO - Starting attempt 2 of 6
[2024-01-30T03:24:08.270+0000] {taskinstance.py:2191} INFO - Executing <Task(SparkSubmitOperator): local_job_raw_to_formatted> on 2024-01-10 00:00:00+00:00
[2024-01-30T03:24:08.280+0000] {standard_task_runner.py:60} INFO - Started process 30299 to run task
[2024-01-30T03:24:08.308+0000] {standard_task_runner.py:87} INFO - Running: ['airflow', 'tasks', 'run', 'manage_data_lake_dag', 'local_job_raw_to_formatted', 'scheduled__2024-01-10T00:00:00+00:00', '--job-id', '177', '--raw', '--subdir', 'DAGS_FOLDER/data_lake_dag.py', '--cfg-path', '/tmp/tmp9uikj86p']
[2024-01-30T03:24:08.311+0000] {standard_task_runner.py:88} INFO - Job 177: Subtask local_job_raw_to_formatted
[2024-01-30T03:24:08.471+0000] {task_command.py:423} INFO - Running <TaskInstance: manage_data_lake_dag.local_job_raw_to_formatted scheduled__2024-01-10T00:00:00+00:00 [running]> on host instance-1.europe-west9-a.c.data-lake-project-409321.internal
[2024-01-30T03:24:08.641+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='juniortemgoua0' AIRFLOW_CTX_DAG_ID='manage_data_lake_dag' AIRFLOW_CTX_TASK_ID='local_job_raw_to_formatted' AIRFLOW_CTX_EXECUTION_DATE='2024-01-10T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='2' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-01-10T00:00:00+00:00'
[2024-01-30T03:24:08.657+0000] {base.py:83} INFO - Using connection ID 'spark_default' for task execution.
[2024-01-30T03:24:08.660+0000] {spark_submit.py:401} INFO - Spark-Submit cmd: spark-submit --master local[*] --conf spark.airflow.execution_date=2024-01-10 --name arrow-spark --class RawToFormatted --queue root.default --deploy-mode client /home/juniortemgoua0/DataLake/jobs/processes/scala/spark_process/target/scala-2.12/spark_job_2.12-0.1.0.jar hdfs://localhost:9000/data-lake
[2024-01-30T03:24:14.934+0000] {spark_submit.py:571} INFO - 24/01/30 03:24:14 INFO SparkContext: Running Spark version 3.5.0
[2024-01-30T03:24:14.945+0000] {spark_submit.py:571} INFO - 24/01/30 03:24:14 INFO SparkContext: OS info Linux, 5.15.0-1049-gcp, amd64
[2024-01-30T03:24:14.947+0000] {spark_submit.py:571} INFO - 24/01/30 03:24:14 INFO SparkContext: Java version 17.0.9
[2024-01-30T03:24:15.319+0000] {spark_submit.py:571} INFO - 24/01/30 03:24:15 WARN NativeCodeLoader: Unable to load native-hadoop library for your platform... using builtin-java classes where applicable
[2024-01-30T03:24:15.821+0000] {spark_submit.py:571} INFO - 24/01/30 03:24:15 INFO ResourceUtils: ==============================================================
[2024-01-30T03:24:15.824+0000] {spark_submit.py:571} INFO - 24/01/30 03:24:15 INFO ResourceUtils: No custom resources configured for spark.driver.
[2024-01-30T03:24:15.826+0000] {spark_submit.py:571} INFO - 24/01/30 03:24:15 INFO ResourceUtils: ==============================================================
[2024-01-30T03:24:15.831+0000] {spark_submit.py:571} INFO - 24/01/30 03:24:15 INFO SparkContext: Submitted application: RawToFormatted
[2024-01-30T03:24:15.944+0000] {spark_submit.py:571} INFO - 24/01/30 03:24:15 INFO ResourceProfile: Default ResourceProfile created, executor resources: Map(cores -> name: cores, amount: 1, script: , vendor: , memory -> name: memory, amount: 1024, script: , vendor: , offHeap -> name: offHeap, amount: 0, script: , vendor: ), task resources: Map(cpus -> name: cpus, amount: 1.0)
[2024-01-30T03:24:15.986+0000] {spark_submit.py:571} INFO - 24/01/30 03:24:15 INFO ResourceProfile: Limiting resource is cpu
[2024-01-30T03:24:15.990+0000] {spark_submit.py:571} INFO - 24/01/30 03:24:15 INFO ResourceProfileManager: Added ResourceProfile id: 0
[2024-01-30T03:24:16.244+0000] {spark_submit.py:571} INFO - 24/01/30 03:24:16 INFO SecurityManager: Changing view acls to: juniortemgoua0
[2024-01-30T03:24:16.249+0000] {spark_submit.py:571} INFO - 24/01/30 03:24:16 INFO SecurityManager: Changing modify acls to: juniortemgoua0
[2024-01-30T03:24:16.253+0000] {spark_submit.py:571} INFO - 24/01/30 03:24:16 INFO SecurityManager: Changing view acls groups to:
[2024-01-30T03:24:16.255+0000] {spark_submit.py:571} INFO - 24/01/30 03:24:16 INFO SecurityManager: Changing modify acls groups to:
[2024-01-30T03:24:16.258+0000] {spark_submit.py:571} INFO - 24/01/30 03:24:16 INFO SecurityManager: SecurityManager: authentication disabled; ui acls disabled; users with view permissions: juniortemgoua0; groups with view permissions: EMPTY; users with modify permissions: juniortemgoua0; groups with modify permissions: EMPTY
[2024-01-30T03:24:17.582+0000] {spark_submit.py:571} INFO - 24/01/30 03:24:17 INFO Utils: Successfully started service 'sparkDriver' on port 33001.
[2024-01-30T03:24:17.754+0000] {spark_submit.py:571} INFO - 24/01/30 03:24:17 INFO SparkEnv: Registering MapOutputTracker
[2024-01-30T03:24:17.886+0000] {spark_submit.py:571} INFO - 24/01/30 03:24:17 INFO SparkEnv: Registering BlockManagerMaster
[2024-01-30T03:24:17.977+0000] {spark_submit.py:571} INFO - 24/01/30 03:24:17 INFO BlockManagerMasterEndpoint: Using org.apache.spark.storage.DefaultTopologyMapper for getting topology information
[2024-01-30T03:24:17.990+0000] {spark_submit.py:571} INFO - 24/01/30 03:24:17 INFO BlockManagerMasterEndpoint: BlockManagerMasterEndpoint up
[2024-01-30T03:24:18.010+0000] {spark_submit.py:571} INFO - 24/01/30 03:24:18 INFO SparkEnv: Registering BlockManagerMasterHeartbeat
[2024-01-30T03:24:18.118+0000] {spark_submit.py:571} INFO - 24/01/30 03:24:18 INFO DiskBlockManager: Created local directory at /tmp/blockmgr-5953793e-17fe-4100-8b1c-fe1d79897407
[2024-01-30T03:24:18.193+0000] {spark_submit.py:571} INFO - 24/01/30 03:24:18 INFO MemoryStore: MemoryStore started with capacity 434.4 MiB
[2024-01-30T03:24:18.271+0000] {spark_submit.py:571} INFO - 24/01/30 03:24:18 INFO SparkEnv: Registering OutputCommitCoordinator
[2024-01-30T03:24:18.903+0000] {spark_submit.py:571} INFO - 24/01/30 03:24:18 INFO JettyUtils: Start Jetty 0.0.0.0:4040 for SparkUI
[2024-01-30T03:24:19.252+0000] {spark_submit.py:571} INFO - 24/01/30 03:24:19 INFO Utils: Successfully started service 'SparkUI' on port 4040.
[2024-01-30T03:24:19.450+0000] {spark_submit.py:571} INFO - 24/01/30 03:24:19 INFO SparkContext: Added JAR file:/home/juniortemgoua0/DataLake/jobs/processes/scala/spark_process/target/scala-2.12/spark_job_2.12-0.1.0.jar at spark://instance-1.europe-west9-a.c.data-lake-project-409321.internal:33001/jars/spark_job_2.12-0.1.0.jar with timestamp 1706585054868
[2024-01-30T03:24:19.835+0000] {spark_submit.py:571} INFO - 24/01/30 03:24:19 INFO Executor: Starting executor ID driver on host instance-1.europe-west9-a.c.data-lake-project-409321.internal
[2024-01-30T03:24:19.836+0000] {spark_submit.py:571} INFO - 24/01/30 03:24:19 INFO Executor: OS info Linux, 5.15.0-1049-gcp, amd64
[2024-01-30T03:24:19.837+0000] {spark_submit.py:571} INFO - 24/01/30 03:24:19 INFO Executor: Java version 17.0.9
[2024-01-30T03:24:19.872+0000] {spark_submit.py:571} INFO - 24/01/30 03:24:19 INFO Executor: Starting executor with user classpath (userClassPathFirst = false): ''
[2024-01-30T03:24:19.879+0000] {spark_submit.py:571} INFO - 24/01/30 03:24:19 INFO Executor: Created or updated repl class loader org.apache.spark.util.MutableURLClassLoader@6ed043d3 for default.
[2024-01-30T03:24:19.948+0000] {spark_submit.py:571} INFO - 24/01/30 03:24:19 INFO Executor: Fetching spark://instance-1.europe-west9-a.c.data-lake-project-409321.internal:33001/jars/spark_job_2.12-0.1.0.jar with timestamp 1706585054868
[2024-01-30T03:24:20.261+0000] {spark_submit.py:571} INFO - 24/01/30 03:24:20 INFO TransportClientFactory: Successfully created connection to instance-1.europe-west9-a.c.data-lake-project-409321.internal/10.200.0.5:33001 after 144 ms (0 ms spent in bootstraps)
[2024-01-30T03:24:20.299+0000] {spark_submit.py:571} INFO - 24/01/30 03:24:20 INFO Utils: Fetching spark://instance-1.europe-west9-a.c.data-lake-project-409321.internal:33001/jars/spark_job_2.12-0.1.0.jar to /tmp/spark-3ea7f11f-18dd-476d-bf7f-73442b1fa74f/userFiles-006103f7-cffe-4902-b604-49cd86a7010e/fetchFileTemp2820019764331274507.tmp
[2024-01-30T03:24:20.374+0000] {spark_submit.py:571} INFO - 24/01/30 03:24:20 INFO Executor: Adding file:/tmp/spark-3ea7f11f-18dd-476d-bf7f-73442b1fa74f/userFiles-006103f7-cffe-4902-b604-49cd86a7010e/spark_job_2.12-0.1.0.jar to class loader default
[2024-01-30T03:24:20.398+0000] {spark_submit.py:571} INFO - 24/01/30 03:24:20 INFO Utils: Successfully started service 'org.apache.spark.network.netty.NettyBlockTransferService' on port 43779.
[2024-01-30T03:24:20.398+0000] {spark_submit.py:571} INFO - 24/01/30 03:24:20 INFO NettyBlockTransferService: Server created on instance-1.europe-west9-a.c.data-lake-project-409321.internal:43779
[2024-01-30T03:24:20.401+0000] {spark_submit.py:571} INFO - 24/01/30 03:24:20 INFO BlockManager: Using org.apache.spark.storage.RandomBlockReplicationPolicy for block replication policy
[2024-01-30T03:24:20.421+0000] {spark_submit.py:571} INFO - 24/01/30 03:24:20 INFO BlockManagerMaster: Registering BlockManager BlockManagerId(driver, instance-1.europe-west9-a.c.data-lake-project-409321.internal, 43779, None)
[2024-01-30T03:24:20.442+0000] {spark_submit.py:571} INFO - 24/01/30 03:24:20 INFO BlockManagerMasterEndpoint: Registering block manager instance-1.europe-west9-a.c.data-lake-project-409321.internal:43779 with 434.4 MiB RAM, BlockManagerId(driver, instance-1.europe-west9-a.c.data-lake-project-409321.internal, 43779, None)
[2024-01-30T03:24:20.446+0000] {spark_submit.py:571} INFO - 24/01/30 03:24:20 INFO BlockManagerMaster: Registered BlockManager BlockManagerId(driver, instance-1.europe-west9-a.c.data-lake-project-409321.internal, 43779, None)
[2024-01-30T03:24:20.447+0000] {spark_submit.py:571} INFO - 24/01/30 03:24:20 INFO BlockManager: Initialized BlockManager: BlockManagerId(driver, instance-1.europe-west9-a.c.data-lake-project-409321.internal, 43779, None)
[2024-01-30T03:24:22.659+0000] {spark_submit.py:571} INFO - 24/01/30 03:24:22 INFO SharedState: Setting hive.metastore.warehouse.dir ('null') to the value of spark.sql.warehouse.dir.
[2024-01-30T03:24:22.688+0000] {spark_submit.py:571} INFO - 24/01/30 03:24:22 INFO SharedState: Warehouse path is 'file:/home/juniortemgoua0/DataLake/spark-warehouse'.
[2024-01-30T03:24:24.322+0000] {spark_submit.py:571} INFO - ************************************************
[2024-01-30T03:24:24.323+0000] {spark_submit.py:571} INFO - Date d'exécution  : 2024-01-10
[2024-01-30T03:24:24.323+0000] {spark_submit.py:571} INFO - ************************************************
[2024-01-30T03:24:24.839+0000] {spark_submit.py:571} INFO - 24/01/30 03:24:24 WARN FileStreamSink: Assume no metadata directory. Error while looking for metadata directory in the path: hdfs://localhost:9000/data-lake/raw/football_data/competitions/2024-01-10/*.json.
[2024-01-30T03:24:24.840+0000] {spark_submit.py:571} INFO - java.net.ConnectException: Call From instance-1/10.200.0.5 to localhost:9000 failed on connection exception: java.net.ConnectException: Connection refused; For more details see:  http://wiki.apache.org/hadoop/ConnectionRefused
[2024-01-30T03:24:24.840+0000] {spark_submit.py:571} INFO - at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
[2024-01-30T03:24:24.840+0000] {spark_submit.py:571} INFO - at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:77)
[2024-01-30T03:24:24.840+0000] {spark_submit.py:571} INFO - at java.base/jdk.internal.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
[2024-01-30T03:24:24.841+0000] {spark_submit.py:571} INFO - at java.base/java.lang.reflect.Constructor.newInstanceWithCaller(Constructor.java:499)
[2024-01-30T03:24:24.841+0000] {spark_submit.py:571} INFO - at java.base/java.lang.reflect.Constructor.newInstance(Constructor.java:480)
[2024-01-30T03:24:24.841+0000] {spark_submit.py:571} INFO - at org.apache.hadoop.net.NetUtils.wrapWithMessage(NetUtils.java:913)
[2024-01-30T03:24:24.841+0000] {spark_submit.py:571} INFO - at org.apache.hadoop.net.NetUtils.wrapException(NetUtils.java:828)
[2024-01-30T03:24:24.841+0000] {spark_submit.py:571} INFO - at org.apache.hadoop.ipc.Client.getRpcResponse(Client.java:1616)
[2024-01-30T03:24:24.841+0000] {spark_submit.py:571} INFO - at org.apache.hadoop.ipc.Client.call(Client.java:1558)
[2024-01-30T03:24:24.842+0000] {spark_submit.py:571} INFO - at org.apache.hadoop.ipc.Client.call(Client.java:1455)
[2024-01-30T03:24:24.842+0000] {spark_submit.py:571} INFO - at org.apache.hadoop.ipc.ProtobufRpcEngine2$Invoker.invoke(ProtobufRpcEngine2.java:242)
[2024-01-30T03:24:24.842+0000] {spark_submit.py:571} INFO - at org.apache.hadoop.ipc.ProtobufRpcEngine2$Invoker.invoke(ProtobufRpcEngine2.java:129)
[2024-01-30T03:24:24.842+0000] {spark_submit.py:571} INFO - at jdk.proxy2/jdk.proxy2.$Proxy34.getFileInfo(Unknown Source)
[2024-01-30T03:24:24.842+0000] {spark_submit.py:571} INFO - at org.apache.hadoop.hdfs.protocolPB.ClientNamenodeProtocolTranslatorPB.getFileInfo(ClientNamenodeProtocolTranslatorPB.java:965)
[2024-01-30T03:24:24.843+0000] {spark_submit.py:571} INFO - at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
[2024-01-30T03:24:24.843+0000] {spark_submit.py:571} INFO - at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
[2024-01-30T03:24:24.843+0000] {spark_submit.py:571} INFO - at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
[2024-01-30T03:24:24.843+0000] {spark_submit.py:571} INFO - at java.base/java.lang.reflect.Method.invoke(Method.java:568)
[2024-01-30T03:24:24.843+0000] {spark_submit.py:571} INFO - at org.apache.hadoop.io.retry.RetryInvocationHandler.invokeMethod(RetryInvocationHandler.java:422)
[2024-01-30T03:24:24.843+0000] {spark_submit.py:571} INFO - at org.apache.hadoop.io.retry.RetryInvocationHandler$Call.invokeMethod(RetryInvocationHandler.java:165)
[2024-01-30T03:24:24.844+0000] {spark_submit.py:571} INFO - at org.apache.hadoop.io.retry.RetryInvocationHandler$Call.invoke(RetryInvocationHandler.java:157)
[2024-01-30T03:24:24.844+0000] {spark_submit.py:571} INFO - at org.apache.hadoop.io.retry.RetryInvocationHandler$Call.invokeOnce(RetryInvocationHandler.java:95)
[2024-01-30T03:24:24.844+0000] {spark_submit.py:571} INFO - at org.apache.hadoop.io.retry.RetryInvocationHandler.invoke(RetryInvocationHandler.java:359)
[2024-01-30T03:24:24.844+0000] {spark_submit.py:571} INFO - at jdk.proxy2/jdk.proxy2.$Proxy35.getFileInfo(Unknown Source)
[2024-01-30T03:24:24.844+0000] {spark_submit.py:571} INFO - at org.apache.hadoop.hdfs.DFSClient.getFileInfo(DFSClient.java:1739)
[2024-01-30T03:24:24.844+0000] {spark_submit.py:571} INFO - at org.apache.hadoop.hdfs.DistributedFileSystem$29.doCall(DistributedFileSystem.java:1753)
[2024-01-30T03:24:24.845+0000] {spark_submit.py:571} INFO - at org.apache.hadoop.hdfs.DistributedFileSystem$29.doCall(DistributedFileSystem.java:1750)
[2024-01-30T03:24:24.845+0000] {spark_submit.py:571} INFO - at org.apache.hadoop.fs.FileSystemLinkResolver.resolve(FileSystemLinkResolver.java:81)
[2024-01-30T03:24:24.845+0000] {spark_submit.py:571} INFO - at org.apache.hadoop.hdfs.DistributedFileSystem.getFileStatus(DistributedFileSystem.java:1765)
[2024-01-30T03:24:24.845+0000] {spark_submit.py:571} INFO - at org.apache.hadoop.fs.FileSystem.isDirectory(FileSystem.java:1777)
[2024-01-30T03:24:24.845+0000] {spark_submit.py:571} INFO - at org.apache.spark.sql.execution.streaming.FileStreamSink$.hasMetadata(FileStreamSink.scala:54)
[2024-01-30T03:24:24.845+0000] {spark_submit.py:571} INFO - at org.apache.spark.sql.execution.datasources.DataSource.resolveRelation(DataSource.scala:366)
[2024-01-30T03:24:24.847+0000] {spark_submit.py:571} INFO - at org.apache.spark.sql.DataFrameReader.loadV1Source(DataFrameReader.scala:229)
[2024-01-30T03:24:24.847+0000] {spark_submit.py:571} INFO - at org.apache.spark.sql.DataFrameReader.$anonfun$load$2(DataFrameReader.scala:211)
[2024-01-30T03:24:24.847+0000] {spark_submit.py:571} INFO - at scala.Option.getOrElse(Option.scala:189)
[2024-01-30T03:24:24.847+0000] {spark_submit.py:571} INFO - at org.apache.spark.sql.DataFrameReader.load(DataFrameReader.scala:211)
[2024-01-30T03:24:24.847+0000] {spark_submit.py:571} INFO - at org.apache.spark.sql.DataFrameReader.json(DataFrameReader.scala:362)
[2024-01-30T03:24:24.848+0000] {spark_submit.py:571} INFO - at org.apache.spark.sql.DataFrameReader.json(DataFrameReader.scala:341)
[2024-01-30T03:24:24.848+0000] {spark_submit.py:571} INFO - at RawToFormatted$.formattedCompetition(RawToFormatted.scala:39)
[2024-01-30T03:24:24.848+0000] {spark_submit.py:571} INFO - at RawToFormatted$.main(RawToFormatted.scala:26)
[2024-01-30T03:24:24.848+0000] {spark_submit.py:571} INFO - at RawToFormatted.main(RawToFormatted.scala)
[2024-01-30T03:24:24.849+0000] {spark_submit.py:571} INFO - at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
[2024-01-30T03:24:24.850+0000] {spark_submit.py:571} INFO - at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
[2024-01-30T03:24:24.850+0000] {spark_submit.py:571} INFO - at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
[2024-01-30T03:24:24.850+0000] {spark_submit.py:571} INFO - at java.base/java.lang.reflect.Method.invoke(Method.java:568)
[2024-01-30T03:24:24.851+0000] {spark_submit.py:571} INFO - at org.apache.spark.deploy.JavaMainApplication.start(SparkApplication.scala:52)
[2024-01-30T03:24:24.851+0000] {spark_submit.py:571} INFO - at org.apache.spark.deploy.SparkSubmit.org$apache$spark$deploy$SparkSubmit$$runMain(SparkSubmit.scala:1029)
[2024-01-30T03:24:24.852+0000] {spark_submit.py:571} INFO - at org.apache.spark.deploy.SparkSubmit.doRunMain$1(SparkSubmit.scala:194)
[2024-01-30T03:24:24.852+0000] {spark_submit.py:571} INFO - at org.apache.spark.deploy.SparkSubmit.submit(SparkSubmit.scala:217)
[2024-01-30T03:24:24.852+0000] {spark_submit.py:571} INFO - at org.apache.spark.deploy.SparkSubmit.doSubmit(SparkSubmit.scala:91)
[2024-01-30T03:24:24.853+0000] {spark_submit.py:571} INFO - at org.apache.spark.deploy.SparkSubmit$$anon$2.doSubmit(SparkSubmit.scala:1120)
[2024-01-30T03:24:24.853+0000] {spark_submit.py:571} INFO - at org.apache.spark.deploy.SparkSubmit$.main(SparkSubmit.scala:1129)
[2024-01-30T03:24:24.853+0000] {spark_submit.py:571} INFO - at org.apache.spark.deploy.SparkSubmit.main(SparkSubmit.scala)
[2024-01-30T03:24:24.854+0000] {spark_submit.py:571} INFO - Caused by: java.net.ConnectException: Connection refused
[2024-01-30T03:24:24.855+0000] {spark_submit.py:571} INFO - at java.base/sun.nio.ch.Net.pollConnect(Native Method)
[2024-01-30T03:24:24.856+0000] {spark_submit.py:571} INFO - at java.base/sun.nio.ch.Net.pollConnectNow(Net.java:672)
[2024-01-30T03:24:24.856+0000] {spark_submit.py:571} INFO - at java.base/sun.nio.ch.SocketChannelImpl.finishConnect(SocketChannelImpl.java:946)
[2024-01-30T03:24:24.856+0000] {spark_submit.py:571} INFO - at org.apache.hadoop.net.SocketIOWithTimeout.connect(SocketIOWithTimeout.java:205)
[2024-01-30T03:24:24.856+0000] {spark_submit.py:571} INFO - at org.apache.hadoop.net.NetUtils.connect(NetUtils.java:586)
[2024-01-30T03:24:24.856+0000] {spark_submit.py:571} INFO - at org.apache.hadoop.ipc.Client$Connection.setupConnection(Client.java:711)
[2024-01-30T03:24:24.858+0000] {spark_submit.py:571} INFO - at org.apache.hadoop.ipc.Client$Connection.setupIOstreams(Client.java:833)
[2024-01-30T03:24:24.858+0000] {spark_submit.py:571} INFO - at org.apache.hadoop.ipc.Client$Connection.access$3800(Client.java:414)
[2024-01-30T03:24:24.858+0000] {spark_submit.py:571} INFO - at org.apache.hadoop.ipc.Client.getConnection(Client.java:1677)
[2024-01-30T03:24:24.858+0000] {spark_submit.py:571} INFO - at org.apache.hadoop.ipc.Client.call(Client.java:1502)
[2024-01-30T03:24:24.858+0000] {spark_submit.py:571} INFO - ... 44 more
[2024-01-30T03:24:24.903+0000] {spark_submit.py:571} INFO - Exception in thread "main" java.net.ConnectException: Call From instance-1/10.200.0.5 to localhost:9000 failed on connection exception: java.net.ConnectException: Connection refused; For more details see:  http://wiki.apache.org/hadoop/ConnectionRefused
[2024-01-30T03:24:24.904+0000] {spark_submit.py:571} INFO - at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
[2024-01-30T03:24:24.904+0000] {spark_submit.py:571} INFO - at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:77)
[2024-01-30T03:24:24.904+0000] {spark_submit.py:571} INFO - at java.base/jdk.internal.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
[2024-01-30T03:24:24.904+0000] {spark_submit.py:571} INFO - at java.base/java.lang.reflect.Constructor.newInstanceWithCaller(Constructor.java:499)
[2024-01-30T03:24:24.904+0000] {spark_submit.py:571} INFO - at java.base/java.lang.reflect.Constructor.newInstance(Constructor.java:480)
[2024-01-30T03:24:24.905+0000] {spark_submit.py:571} INFO - at org.apache.hadoop.net.NetUtils.wrapWithMessage(NetUtils.java:913)
[2024-01-30T03:24:24.905+0000] {spark_submit.py:571} INFO - at org.apache.hadoop.net.NetUtils.wrapException(NetUtils.java:828)
[2024-01-30T03:24:24.905+0000] {spark_submit.py:571} INFO - at org.apache.hadoop.ipc.Client.getRpcResponse(Client.java:1616)
[2024-01-30T03:24:24.905+0000] {spark_submit.py:571} INFO - at org.apache.hadoop.ipc.Client.call(Client.java:1558)
[2024-01-30T03:24:24.905+0000] {spark_submit.py:571} INFO - at org.apache.hadoop.ipc.Client.call(Client.java:1455)
[2024-01-30T03:24:24.906+0000] {spark_submit.py:571} INFO - at org.apache.hadoop.ipc.ProtobufRpcEngine2$Invoker.invoke(ProtobufRpcEngine2.java:242)
[2024-01-30T03:24:24.906+0000] {spark_submit.py:571} INFO - at org.apache.hadoop.ipc.ProtobufRpcEngine2$Invoker.invoke(ProtobufRpcEngine2.java:129)
[2024-01-30T03:24:24.906+0000] {spark_submit.py:571} INFO - at jdk.proxy2/jdk.proxy2.$Proxy34.getListing(Unknown Source)
[2024-01-30T03:24:24.906+0000] {spark_submit.py:571} INFO - at org.apache.hadoop.hdfs.protocolPB.ClientNamenodeProtocolTranslatorPB.getListing(ClientNamenodeProtocolTranslatorPB.java:688)
[2024-01-30T03:24:24.909+0000] {spark_submit.py:571} INFO - at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
[2024-01-30T03:24:24.909+0000] {spark_submit.py:571} INFO - at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
[2024-01-30T03:24:24.909+0000] {spark_submit.py:571} INFO - at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
[2024-01-30T03:24:24.909+0000] {spark_submit.py:571} INFO - at java.base/java.lang.reflect.Method.invoke(Method.java:568)
[2024-01-30T03:24:24.910+0000] {spark_submit.py:571} INFO - at org.apache.hadoop.io.retry.RetryInvocationHandler.invokeMethod(RetryInvocationHandler.java:422)
[2024-01-30T03:24:24.910+0000] {spark_submit.py:571} INFO - at org.apache.hadoop.io.retry.RetryInvocationHandler$Call.invokeMethod(RetryInvocationHandler.java:165)
[2024-01-30T03:24:24.910+0000] {spark_submit.py:571} INFO - at org.apache.hadoop.io.retry.RetryInvocationHandler$Call.invoke(RetryInvocationHandler.java:157)
[2024-01-30T03:24:24.910+0000] {spark_submit.py:571} INFO - at org.apache.hadoop.io.retry.RetryInvocationHandler$Call.invokeOnce(RetryInvocationHandler.java:95)
[2024-01-30T03:24:24.910+0000] {spark_submit.py:571} INFO - at org.apache.hadoop.io.retry.RetryInvocationHandler.invoke(RetryInvocationHandler.java:359)
[2024-01-30T03:24:24.911+0000] {spark_submit.py:571} INFO - at jdk.proxy2/jdk.proxy2.$Proxy35.getListing(Unknown Source)
[2024-01-30T03:24:24.911+0000] {spark_submit.py:571} INFO - at org.apache.hadoop.hdfs.DFSClient.listPaths(DFSClient.java:1702)
[2024-01-30T03:24:24.911+0000] {spark_submit.py:571} INFO - at org.apache.hadoop.hdfs.DFSClient.listPaths(DFSClient.java:1686)
[2024-01-30T03:24:24.911+0000] {spark_submit.py:571} INFO - at org.apache.hadoop.hdfs.DistributedFileSystem.listStatusInternal(DistributedFileSystem.java:1100)
[2024-01-30T03:24:24.912+0000] {spark_submit.py:571} INFO - at org.apache.hadoop.hdfs.DistributedFileSystem.access$600(DistributedFileSystem.java:147)
[2024-01-30T03:24:24.912+0000] {spark_submit.py:571} INFO - at org.apache.hadoop.hdfs.DistributedFileSystem$24.doCall(DistributedFileSystem.java:1175)
[2024-01-30T03:24:24.912+0000] {spark_submit.py:571} INFO - at org.apache.hadoop.hdfs.DistributedFileSystem$24.doCall(DistributedFileSystem.java:1172)
[2024-01-30T03:24:24.912+0000] {spark_submit.py:571} INFO - at org.apache.hadoop.fs.FileSystemLinkResolver.resolve(FileSystemLinkResolver.java:81)
[2024-01-30T03:24:24.912+0000] {spark_submit.py:571} INFO - at org.apache.hadoop.hdfs.DistributedFileSystem.listStatus(DistributedFileSystem.java:1182)
[2024-01-30T03:24:24.913+0000] {spark_submit.py:571} INFO - at org.apache.hadoop.fs.Globber.listStatus(Globber.java:128)
[2024-01-30T03:24:24.913+0000] {spark_submit.py:571} INFO - at org.apache.hadoop.fs.Globber.doGlob(Globber.java:291)
[2024-01-30T03:24:24.913+0000] {spark_submit.py:571} INFO - at org.apache.hadoop.fs.Globber.glob(Globber.java:202)
[2024-01-30T03:24:24.913+0000] {spark_submit.py:571} INFO - at org.apache.hadoop.fs.FileSystem.globStatus(FileSystem.java:2124)
[2024-01-30T03:24:24.913+0000] {spark_submit.py:571} INFO - at org.apache.spark.deploy.SparkHadoopUtil.globPath(SparkHadoopUtil.scala:238)
[2024-01-30T03:24:24.914+0000] {spark_submit.py:571} INFO - at org.apache.spark.sql.execution.datasources.DataSource$.$anonfun$checkAndGlobPathIfNecessary$3(DataSource.scala:737)
[2024-01-30T03:24:24.914+0000] {spark_submit.py:571} INFO - at org.apache.spark.util.ThreadUtils$.$anonfun$parmap$2(ThreadUtils.scala:380)
[2024-01-30T03:24:24.914+0000] {spark_submit.py:571} INFO - at scala.concurrent.Future$.$anonfun$apply$1(Future.scala:659)
[2024-01-30T03:24:24.914+0000] {spark_submit.py:571} INFO - at scala.util.Success.$anonfun$map$1(Try.scala:255)
[2024-01-30T03:24:24.914+0000] {spark_submit.py:571} INFO - at scala.util.Success.map(Try.scala:213)
[2024-01-30T03:24:24.914+0000] {spark_submit.py:571} INFO - at scala.concurrent.Future.$anonfun$map$1(Future.scala:292)
[2024-01-30T03:24:24.915+0000] {spark_submit.py:571} INFO - at scala.concurrent.impl.Promise.liftedTree1$1(Promise.scala:33)
[2024-01-30T03:24:24.915+0000] {spark_submit.py:571} INFO - at scala.concurrent.impl.Promise.$anonfun$transform$1(Promise.scala:33)
[2024-01-30T03:24:24.915+0000] {spark_submit.py:571} INFO - at scala.concurrent.impl.CallbackRunnable.run(Promise.scala:64)
[2024-01-30T03:24:24.916+0000] {spark_submit.py:571} INFO - at java.base/java.util.concurrent.ForkJoinTask$RunnableExecuteAction.exec(ForkJoinTask.java:1395)
[2024-01-30T03:24:24.916+0000] {spark_submit.py:571} INFO - at java.base/java.util.concurrent.ForkJoinTask.doExec(ForkJoinTask.java:373)
[2024-01-30T03:24:24.916+0000] {spark_submit.py:571} INFO - at java.base/java.util.concurrent.ForkJoinPool$WorkQueue.topLevelExec(ForkJoinPool.java:1182)
[2024-01-30T03:24:24.916+0000] {spark_submit.py:571} INFO - at java.base/java.util.concurrent.ForkJoinPool.scan(ForkJoinPool.java:1655)
[2024-01-30T03:24:24.916+0000] {spark_submit.py:571} INFO - at java.base/java.util.concurrent.ForkJoinPool.runWorker(ForkJoinPool.java:1622)
[2024-01-30T03:24:24.916+0000] {spark_submit.py:571} INFO - at java.base/java.util.concurrent.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:165)
[2024-01-30T03:24:24.917+0000] {spark_submit.py:571} INFO - Caused by: java.net.ConnectException: Connection refused
[2024-01-30T03:24:24.917+0000] {spark_submit.py:571} INFO - at java.base/sun.nio.ch.Net.pollConnect(Native Method)
[2024-01-30T03:24:24.917+0000] {spark_submit.py:571} INFO - at java.base/sun.nio.ch.Net.pollConnectNow(Net.java:672)
[2024-01-30T03:24:24.917+0000] {spark_submit.py:571} INFO - at java.base/sun.nio.ch.SocketChannelImpl.finishConnect(SocketChannelImpl.java:946)
[2024-01-30T03:24:24.919+0000] {spark_submit.py:571} INFO - at org.apache.hadoop.net.SocketIOWithTimeout.connect(SocketIOWithTimeout.java:205)
[2024-01-30T03:24:24.919+0000] {spark_submit.py:571} INFO - at org.apache.hadoop.net.NetUtils.connect(NetUtils.java:586)
[2024-01-30T03:24:24.919+0000] {spark_submit.py:571} INFO - at org.apache.hadoop.ipc.Client$Connection.setupConnection(Client.java:711)
[2024-01-30T03:24:24.919+0000] {spark_submit.py:571} INFO - at org.apache.hadoop.ipc.Client$Connection.setupIOstreams(Client.java:833)
[2024-01-30T03:24:24.919+0000] {spark_submit.py:571} INFO - at org.apache.hadoop.ipc.Client$Connection.access$3800(Client.java:414)
[2024-01-30T03:24:24.919+0000] {spark_submit.py:571} INFO - at org.apache.hadoop.ipc.Client.getConnection(Client.java:1677)
[2024-01-30T03:24:24.920+0000] {spark_submit.py:571} INFO - at org.apache.hadoop.ipc.Client.call(Client.java:1502)
[2024-01-30T03:24:24.920+0000] {spark_submit.py:571} INFO - ... 43 more
[2024-01-30T03:24:24.925+0000] {spark_submit.py:571} INFO - 24/01/30 03:24:24 INFO SparkContext: Invoking stop() from shutdown hook
[2024-01-30T03:24:24.926+0000] {spark_submit.py:571} INFO - 24/01/30 03:24:24 INFO SparkContext: SparkContext is stopping with exitCode 0.
[2024-01-30T03:24:24.941+0000] {spark_submit.py:571} INFO - 24/01/30 03:24:24 INFO SparkUI: Stopped Spark web UI at http://instance-1.europe-west9-a.c.data-lake-project-409321.internal:4040
[2024-01-30T03:24:24.982+0000] {spark_submit.py:571} INFO - 24/01/30 03:24:24 INFO MapOutputTrackerMasterEndpoint: MapOutputTrackerMasterEndpoint stopped!
[2024-01-30T03:24:25.000+0000] {spark_submit.py:571} INFO - 24/01/30 03:24:24 INFO MemoryStore: MemoryStore cleared
[2024-01-30T03:24:25.001+0000] {spark_submit.py:571} INFO - 24/01/30 03:24:25 INFO BlockManager: BlockManager stopped
[2024-01-30T03:24:25.014+0000] {spark_submit.py:571} INFO - 24/01/30 03:24:25 INFO BlockManagerMaster: BlockManagerMaster stopped
[2024-01-30T03:24:25.020+0000] {spark_submit.py:571} INFO - 24/01/30 03:24:25 INFO OutputCommitCoordinator$OutputCommitCoordinatorEndpoint: OutputCommitCoordinator stopped!
[2024-01-30T03:24:25.030+0000] {spark_submit.py:571} INFO - 24/01/30 03:24:25 INFO SparkContext: Successfully stopped SparkContext
[2024-01-30T03:24:25.031+0000] {spark_submit.py:571} INFO - 24/01/30 03:24:25 INFO ShutdownHookManager: Shutdown hook called
[2024-01-30T03:24:25.031+0000] {spark_submit.py:571} INFO - 24/01/30 03:24:25 INFO ShutdownHookManager: Deleting directory /tmp/spark-3ea7f11f-18dd-476d-bf7f-73442b1fa74f
[2024-01-30T03:24:25.036+0000] {spark_submit.py:571} INFO - 24/01/30 03:24:25 INFO ShutdownHookManager: Deleting directory /tmp/spark-025260b3-bc4e-4b15-8eb8-901b5f331e22
[2024-01-30T03:24:25.104+0000] {taskinstance.py:2698} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/juniortemgoua0/DataLake/venv/lib/python3.8/site-packages/airflow/models/taskinstance.py", line 433, in _execute_task
    result = execute_callable(context=context, **execute_callable_kwargs)
  File "/home/juniortemgoua0/DataLake/venv/lib/python3.8/site-packages/airflow/providers/apache/spark/operators/spark_submit.py", line 174, in execute
    self._hook.submit(self._application)
  File "/home/juniortemgoua0/DataLake/venv/lib/python3.8/site-packages/airflow/providers/apache/spark/hooks/spark_submit.py", line 502, in submit
    raise AirflowException(
airflow.exceptions.AirflowException: Cannot execute: spark-submit --master local[*] --conf spark.airflow.execution_date=2024-01-10 --name arrow-spark --class RawToFormatted --queue root.default --deploy-mode client /home/juniortemgoua0/DataLake/jobs/processes/scala/spark_process/target/scala-2.12/spark_job_2.12-0.1.0.jar hdfs://localhost:9000/data-lake. Error code is: 1.
[2024-01-30T03:24:25.107+0000] {taskinstance.py:1138} INFO - Marking task as UP_FOR_RETRY. dag_id=manage_data_lake_dag, task_id=local_job_raw_to_formatted, execution_date=20240110T000000, start_date=20240130T032408, end_date=20240130T032425
[2024-01-30T03:24:25.120+0000] {standard_task_runner.py:107} ERROR - Failed to execute job 177 for task local_job_raw_to_formatted (Cannot execute: spark-submit --master local[*] --conf spark.airflow.execution_date=2024-01-10 --name arrow-spark --class RawToFormatted --queue root.default --deploy-mode client /home/juniortemgoua0/DataLake/jobs/processes/scala/spark_process/target/scala-2.12/spark_job_2.12-0.1.0.jar hdfs://localhost:9000/data-lake. Error code is: 1.; 30299)
[2024-01-30T03:24:25.164+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 1
[2024-01-30T03:24:25.180+0000] {taskinstance.py:3280} INFO - 0 downstream tasks scheduled from follow-on schedule check
